auth/jwt/middleware.go

```diff
@@ -79,10 +79,10 @@ func MapClaimsFactory() jwt.Claims {
 	return jwt.MapClaims{}
 }
 
-// StandardClaimsFactory is a ClaimsFactory that returns
-// an empty jwt.StandardClaims.
-func StandardClaimsFactory() jwt.Claims {
-	return &jwt.StandardClaims{}
+// RegisteredClaimsFactory is a ClaimsFactory that returns
+// an empty jwt.RegisteredClaims.
+func RegisteredClaimsFactory() jwt.Claims {
+	return &jwt.RegisteredClaims{}
```

Comment:
In case naming consistency is not so critical, we can consider to keep `StandardClaimsFactory` so to avoid a breaking change in go kit.
From my understanding `jwt.RegisteredClaims{}` is mostly backwards compatible with `jwt.StandardClaims{}`.

Any feedback is appreciated

Reply:
Changing public API identifiers like this should probably be avoided if at all possible.